lib/tasks/plugin.rake

```diff
@@ -88,6 +88,45 @@ task 'plugin:update', :plugin do |t, args|
   abort('Unable to pull latest version of plugin') unless update_status
 end
 
+desc 'checkout all compatible plugin versions'
+task 'plugin:checkout_compatible_all' do |t|
```

Comment:
I suggest we change the name of the tasks to `plugin:pull_compatible` - because it's already been checked out and this is about upgrading the plugin.